assets/javascripts/discourse/connectors/topic-above-posts/discourse-templates-button.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="template-topic-controls">
+  <DButton
+    @icon={{if this.copyConfirm "check" "copy"}}
+    @class={{concat-class
+      "btn-default"
+      "template-copy"
+      (if this.copyConfirm "ok")
```

Comment:
It would be cool to have this as a general `DButton` option. For actions like copying, where since there are no route changes, the button itself changes to show confirmation. And I guess after a while it would need to revert back to the default state as well. 

(Separate discussion, I know, this just gave me an idea.)